definitions/checks/pulpcore/no_running_tasks.rb

```diff
@@ -0,0 +1,32 @@
+module Checks::Pulpcore
+  class NoRunningTasks < ForemanMaintain::Check
+    metadata do
+      for_feature :pulpcore
+      description 'Check for running pulpcore tasks'
+      tags :pre_upgrade
```

Comment:
Unnecessary now.

Reply:
Added it to the scenarios.

Should all other checks loose their `tags` entry too?

Reply:
Yes - but we can follow up with that.

Reply:
https://github.com/theforeman/foreman_maintain/issues/903